man/org.freedesktop.systemd1.xml

```diff
@@ -171,6 +171,7 @@ node /org/freedesktop/systemd1 {
       DumpByFileDescriptor(out h fd);
       DumpUnitsMatchingPatternsByFileDescriptor(in  as patterns,
                                                 out h fd);
+      DumpMemoryStateByFileDescriptor(out h fd);
```

Comment:
memory state?

it's about allocation, not the memory itself. maybe.

DumpMemoryAllocationStateByFileDescriptor()